### DISCUSSION TITLE
How can I uninstall this package?

### DISCUSSION THREAD

**Discussion by diksown** [insidegui/ocrit#2] [Q&A] [answered]
The title is self descriptive. I installed it using the .pkg file in the release (0.1.0), but I don't know how to uninstall it.

I searched about how to do it and looks like there isn't a single solution for the .pkg format.

 I saw a recommendation to [contact the developers of the app](https://apple.stackexchange.com/a/383738) in this case. I couldn't find a issues page here, so I'm asking on this discussions page.

Could someone kindly help me?

**Comment by insidegui** [comment, marked as answer]
There's only a single file that gets installed by this package, so it can be deleted manually.

You may delete it manually like so:

### Uninstall Using Terminal

Run `sudo rm /usr/local/bin/ocrit` in Terminal

### Uninstall Using Finder

- Go to Finder
- Select the "Go" menu
- Click "Go to Folder"
- Paste `/usr/local/bin`
- Find and select the `ocrit` file
- Delete it with Command + Delete (you may be asked for a password or Touch ID)

**Reply by diksown** [reply]
Thank you so much for the answer (and for the app, btw)! 🇧🇷
